packages/fiori/src/MediaGalleryItemTemplate.tsx

```diff
@@ -0,0 +1,21 @@
+import type MediaGalleryItem from "./MediaGalleryItem.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+
+export default function MediaGalleryItemTemplate(this: MediaGalleryItem) {
+	return (
+		<div class="ui5-media-gallery-item-root"
+			tabindex={this.effectiveTabIndex}
+			data-sap-focus-ref
+			onKeyDown={this._onkeydown}
+			onKeyUp={this._onkeyup}
+			role={this._role}
+		>
+			<div class="ui5-media-gallery-item-mask-layer"></div>
+			<div class="ui5-media-gallery-item-wrapper" style={this.styles.wrapper}>
+				{this._showBackgroundIcon && <Icon name="background"></Icon>}
```

Comment:
One new thing around the migration is that we also encourage importing the used icons in the template
and make use of its default export to bind it:
```ts
import background from "@ui5/webcomponents-icons/dist/background.js";

<Icon name={background}></Icon>
```

This way we will avoid icons missed to be imported, because using the hardcoded string does not have relation to the import.